### PR TITLE
[Remove Stream/Optional usage] LPS-170768 dynamic-data-mapping | PART 3

### PR DESCRIPTION
**Pull request by JanisZhang**
This PR is to Remove usages of Stream/Optional on Method Return Value for module: dynamic-data-mapping. Thanks!

### CODE REVIEW

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ConcatFunction.java [hunk 1]
```diff
@@ -30,15 +27,15 @@
...
-		return Stream.of(
-			values
-		).filter(
-			Objects::nonNull
-		).map(
-			Object::toString
-		).collect(
-			Collectors.joining()
-		);
+		StringBundler sb = new StringBundler();
```
[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ConcatFunction.java:30]: thanks!

**Comment by JanisZhang** [comment]
force-pushed. thanks! @dantewang @tinatian

**Comment by dantewang** [comment]
ci:test:relevant

**Pull request closed without merging**